ctx.go

```diff
@@ -43,5 +43,8 @@ func Ctx(ctx context.Context) *Logger {
 	if l, ok := ctx.Value(ctxKey{}).(*Logger); ok {
 		return l
 	}
+	if StrictMode {
```

Comment:
`StrictMode` is two broad of a name.

Reply:
I can agree with that - any suggestions? I would go with `UseDefaultLogger bool` - is it good enough?